src/schema/comments.ts

```diff
@@ -770,11 +779,13 @@ export const resolvers: IResolvers<any, Context> = {
         const squadId =
           source.type === SourceType.Squad ? source.id : undefined;
         const comment = await ctx.con.transaction(async (entityManager) => {
+          const commentId = await generateShortId();
           const createdComment = entityManager.getRepository(Comment).create({
-            id: await generateShortId(),
+            id: commentId,
             postId,
             userId: ctx.userId,
             content,
+            banned: await autobanComment(commentId, ctx),
```

Comment:
I think in `ensureSourcePermissions` we already fetch the user. Can you check and reuse the same object if possible?

Reply:
Nope, we fetch SourceMember, not user.